### PR TITLE
docs: Added bouncing ball example in collision_detection examples and updated flame_forge2d to latest

### PR DESCRIPTION
**Pull request by rutvik110**
## Description

This PR adds two updates.

1. It updates the flame_forge2d to latest version i.e on pub.dev in examples and padracing projects. 
2. Adds a bouncing ball example in the collision detection examples demonstrating a simple example of collision and bouncing of a ball from the walls around it.

### Preview of the bouncing ball example added
https://user-images.githubusercontent.com/65209850/187041471-77635c70-b75a-4db6-b692-62899014e54f.mov

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `docs:` etc).
- [x] I have read the [Contributor Guide] and followed the process outlined for submitting PRs.
- [x] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation in `docs` and added dartdoc comments with `///`.
- [x] I have updated/added relevant examples in `examples`.

## Breaking Change

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

### CODE REVIEW

File: examples/lib/stories/collision_detection/bouncing_ball_example.dart [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    This example shows how you can use the Collisions detection api to know when a ball
+    collides with the walls we set up and then update it to bounce off these walls.
+  ''';
+  @override
+  Future<void>? onLoad() {
+    final boundaries = createBoundaries();
+
+    boundaries.forEach(add);
+
+    addAll([
```
Comment (spydon) [inline examples/lib/stories/collision_detection/bouncing_ball_example.dart:20]: `add` is enough here

File: examples/lib/stories/collision_detection/bouncing_ball_example.dart [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+        return Vector2(5, game.size.y);
+      case BoundarySide.right:
+        return Vector2(5, game.size.y);
+      default:
+        return Vector2(0, 0);
+    }
+  }
+}
+
+List<Boundary> createBoundaries() {
```
Comment (spydon) [inline examples/lib/stories/collision_detection/bouncing_ball_example.dart:149]: Is this needed? Can't the `ScreenHitbox` be used?

**Review by spydon — COMMENTED** [review]
Just some small comments, thanks for the PR!

File: examples/lib/stories/collision_detection/bouncing_ball_example.dart (hunk 1 above)
Comment (rutvik110) [reply examples/lib/stories/collision_detection/bouncing_ball_example.dart:20]: yah, I'll update that!

File: examples/lib/stories/collision_detection/bouncing_ball_example.dart (hunk 2 above)
Comment (rutvik110) [reply examples/lib/stories/collision_detection/bouncing_ball_example.dart:149]: Right! I actually had that in my existing project so it just slipped in. Will update that with the `ScreenHitBox`

**Comment by rutvik110** [comment]
Hey @spydon , made the changes you asked for.🙌

**Review by spydon — APPROVED** [review]
Lgtm, nice with so much less code with the `ScreenHitbox`

**Comment by rutvik110** [comment]
> Lgtm, nice with so much less code with the `ScreenHitbox`

Exactly! 😅
Thanx for the reminder!

**Pull request merged**